packages/turf-clusters-dbscan/index.ts

```diff
@@ -121,14 +120,16 @@ function clustersDbscan(
 
     // Calculate the bounding box for the region query
     const bbox = { minX, minY, maxX, maxY };
-    return tree.search(bbox).filter((neighbor: IndexedPoint) => {
-      const neighborIndex = neighbor.index;
-      const neighborPoint = points.features[neighborIndex];
-      const distanceInKm = distance(point, neighborPoint, {
-        units: "kilometers",
-      });
-      return distanceInKm <= maxDistance;
-    }) as IndexedPoint[];
+    return (tree.search(bbox) as ReadonlyArray<IndexedPoint>).filter(
+      (neighbor) => {
+        const neighborIndex = neighbor.index;
+        const neighborPoint = points.features[neighborIndex];
+        const distanceInKm = distance(point, neighborPoint, {
+          units: "kilometers",
+        });
+        return distanceInKm <= maxDistance;
+      }
+    );
```

Comment:
Had to make changes in this file to get the build to pass? Not sure what's up with that.

Reply:
Using @types/rbush in future may help resolve this